src/Akka.HealthCheck/AkkaHealthCheck.cs

```diff
@@ -102,16 +102,19 @@ public AkkaHealthCheck(HealthCheckSettings settings, ExtendedActorSystem system)
         public static IActorRef StartTransportActor(ITransportSettings settings, ExtendedActorSystem system,
             ProbeKind probeKind, IActorRef probe)
         {
+            var  _log = system.Log;
             if (settings is FileTransportSettings fileTransport)
                 switch (probeKind)
                 {
                     case ProbeKind.Liveness:
+                        _log.Debug("Liveness file transport created");
```

Comment:
Need to log the actual settings - where is the file being written, which network port is being opened, etc...